include/dxc/Support/dxcapi.extval.h

```diff
@@ -42,7 +42,7 @@ class DxcDllExtValidationLoader : public DllLoader {
 
   bool IsEnabled() const override { return DxCompilerSupport.IsEnabled(); }
 
-  void Cleanup() override {
+  void Cleanup() {
     DxilExtValSupport.Cleanup();
     DxCompilerSupport.Cleanup();
   }
```

Comment:
Is this actually used by anything? The point of my feedback about removing this from the base class is that Cleanup is only ever called on SpecificDllLoader.

Reply:
Ah right, then yeah I should be able to remove this entirely.